2019-12-29 08:45:47,797 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert24\app\__init__.py:48]
2019-12-29 08:50:23,981 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert24\app\__init__.py:48]
2019-12-29 09:34:51,386 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert24\app\__init__.py:48]
2019-12-29 09:57:47,540 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert24\app\__init__.py:48]
2019-12-29 10:11:05,108 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert24\app\__init__.py:48]
2019-12-29 10:12:15,326 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert24\app\__init__.py:48]
2019-12-29 10:15:30,696 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert24\app\__init__.py:48]
2019-12-29 10:18:21,996 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert24\app\__init__.py:48]
2019-12-29 10:19:21,781 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert24\app\__init__.py:48]
2019-12-29 12:21:59,157 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert24\app\__init__.py:48]
2019-12-29 12:26:48,601 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert24\app\__init__.py:48]
2019-12-29 12:43:48,365 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert24\app\__init__.py:48]
2019-12-30 08:46:59,599 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert24\app\__init__.py:48]
2019-12-30 09:07:41,640 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert24\app\__init__.py:48]
2019-12-30 09:09:22,416 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert24\app\__init__.py:48]
2019-12-30 09:12:37,483 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert24\app\__init__.py:48]
2019-12-30 09:17:58,674 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert24\app\__init__.py:48]
2019-12-30 15:24:36,918 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert24\app\__init__.py:48]
2019-12-31 07:44:13,089 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert24\app\__init__.py:48]
2019-12-31 07:57:50,828 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert24\app\__init__.py:48]
2019-12-31 08:08:47,228 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert24\app\__init__.py:48]
2019-12-31 08:56:43,046 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert24\app\__init__.py:48]
2019-12-31 09:07:08,832 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert24\app\__init__.py:48]
2019-12-31 09:09:20,963 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert24\app\__init__.py:48]
2019-12-31 09:15:24,480 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert24\app\__init__.py:48]
2019-12-31 09:17:12,978 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert24\app\__init__.py:48]
2019-12-31 09:21:09,790 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert24\app\__init__.py:48]
2019-12-31 09:23:18,611 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert24\app\__init__.py:48]
2020-01-01 10:41:31,234 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert25\app\__init__.py:48]
2020-01-01 10:59:14,861 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert25\app\__init__.py:48]
2020-01-01 11:10:48,972 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert25\app\__init__.py:48]
2020-01-01 12:08:55,478 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert25\app\__init__.py:48]
2020-01-01 12:14:32,379 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert25\app\__init__.py:48]
2020-01-01 12:21:07,484 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert25\app\__init__.py:48]
2020-01-01 12:24:48,039 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert25\app\__init__.py:48]
2020-01-01 12:48:54,395 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert25\app\__init__.py:48]
2020-01-01 12:50:28,775 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert25\app\__init__.py:48]
2020-01-01 12:53:40,749 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert25\app\__init__.py:48]
2020-01-01 13:13:35,498 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert25\app\__init__.py:48]
2020-01-01 14:51:55,678 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert25\app\__init__.py:48]
2020-01-01 15:10:32,024 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert25\app\__init__.py:48]
2020-01-01 15:12:12,464 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert25\app\__init__.py:48]
2020-01-01 15:13:02,435 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert25\app\__init__.py:48]
2020-01-01 15:13:10,733 ERROR: Exception on / [GET] [in c:\Users\richa\OneDrive\Projects\Cert25\venv\lib\site-packages\flask\app.py:1780]
Traceback (most recent call last):
  File "c:\Users\richa\OneDrive\Projects\Cert25\venv\lib\site-packages\flask\app.py", line 2311, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\Users\richa\OneDrive\Projects\Cert25\venv\lib\site-packages\flask\app.py", line 1834, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\Users\richa\OneDrive\Projects\Cert25\venv\lib\site-packages\flask\app.py", line 1737, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\Users\richa\OneDrive\Projects\Cert25\venv\lib\site-packages\flask\_compat.py", line 36, in reraise
    raise value
  File "c:\Users\richa\OneDrive\Projects\Cert25\venv\lib\site-packages\flask\app.py", line 1832, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\Users\richa\OneDrive\Projects\Cert25\venv\lib\site-packages\flask\app.py", line 1818, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\Users\richa\OneDrive\Projects\Cert25\app\routes.py", line 70, in home
    certifiedShop2=certifiedShop2,notCertifiedShop1=notCertifiedShop1,notCertifiedShop2=notCertifiedShop2,noCertification=noCertification)
  File "c:\Users\richa\OneDrive\Projects\Cert25\venv\lib\site-packages\flask\templating.py", line 135, in render_template
    context, ctx.app)
  File "c:\Users\richa\OneDrive\Projects\Cert25\venv\lib\site-packages\flask\templating.py", line 117, in _render
    rv = template.render(context)
  File "c:\Users\richa\OneDrive\Projects\Cert25\venv\lib\site-packages\jinja2\asyncsupport.py", line 76, in render
    return original_render(self, *args, **kwargs)
  File "c:\Users\richa\OneDrive\Projects\Cert25\venv\lib\site-packages\jinja2\environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "c:\Users\richa\OneDrive\Projects\Cert25\venv\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "c:\Users\richa\OneDrive\Projects\Cert25\venv\lib\site-packages\jinja2\_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "c:\Users\richa\OneDrive\Projects\Cert25\app\templates\home.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "c:\Users\richa\OneDrive\Projects\Cert25\app\templates\base.html", line 1, in top-level template code
    {% extends 'bootstrap/base.html' %}
  File "c:\Users\richa\OneDrive\Projects\Cert25\app\templates\bootstrap\base.html", line 1, in top-level template code
    {% block doc -%}
  File "c:\Users\richa\OneDrive\Projects\Cert25\app\templates\bootstrap\base.html", line 4, in block "doc"
    {%- block html %}
  File "c:\Users\richa\OneDrive\Projects\Cert25\app\templates\bootstrap\base.html", line 33, in block "html"
    {% block body -%}
  File "c:\Users\richa\OneDrive\Projects\Cert25\app\templates\bootstrap\base.html", line 34, in block "body"
    {% block navbar %}
  File "c:\Users\richa\OneDrive\Projects\Cert25\app\templates\base.html", line 33, in block "navbar"
    <a class="nav-link" href="{{ url_for('notcertified') }}">NOT CERTIFIED</a>
  File "c:\Users\richa\OneDrive\Projects\Cert25\venv\lib\site-packages\flask\helpers.py", line 356, in url_for
    return appctx.app.handle_url_build_error(error, endpoint, values)
  File "c:\Users\richa\OneDrive\Projects\Cert25\venv\lib\site-packages\flask\app.py", line 2080, in handle_url_build_error
    reraise(exc_type, exc_value, tb)
  File "c:\Users\richa\OneDrive\Projects\Cert25\venv\lib\site-packages\flask\_compat.py", line 36, in reraise
    raise value
  File "c:\Users\richa\OneDrive\Projects\Cert25\venv\lib\site-packages\flask\helpers.py", line 345, in url_for
    force_external=external)
  File "c:\Users\richa\OneDrive\Projects\Cert25\venv\lib\site-packages\werkzeug\routing.py", line 2007, in build
    raise BuildError(endpoint, values, method, self)
werkzeug.routing.BuildError: Could not build url for endpoint 'notcertified'. Did you mean 'certify' instead?
2020-01-01 15:14:20,809 INFO: Certification startup [in c:\Users\richa\OneDrive\Projects\Cert25\app\__init__.py:48]
